_layouts/user-guide.html

```diff
@@ -7,6 +7,9 @@
                 <div class="info-block text-adapt">
                     <div>
                         {{content}}
+                        <p style="text-align: right;">

```

Comment:
Generally not a fan of inline CSS styling, but I've been known to use it for little things. Other than this quibble, it's good to merge.